hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiPropertySource.java

```diff
@@ -297,6 +298,10 @@ static FileID asFile(String v) {
                 .build();
     }
 
+    static EntityNumber asNode(String v) {
```

Comment:
Can probably rename as `asEntityNumber()` to be consistent with similar methods like `asAccount`

Reply:
Done